sim/src/main/makefrag/firesim/Makefrag

```diff
@@ -191,7 +191,7 @@ vcs = $(GENERATED_DIR)/$(DESIGN)
 vcs_debug = $(GENERATED_DIR)/$(DESIGN)-debug
 vcs_args = +vcs+initreg+0 +vcs+initmem+0
 xsim = $(GENERATED_DIR)/$(DESIGN)-$(PLATFORM)
-sim_binary_basename := $(basename $(notdir $(SIM_BINARY)))
+sim_binary_basename = $(basename $(notdir $(SIM_BINARY)))
```

Comment:
change to recursive type so that --warn-undefined-variables doesn't complain about SIM_BINARY.  It is expected to be given on the cmdline.  It might be nice in the future to do an explicit check that if one of the `run-$EMUL-$DEBUG` targets is being run, that SIM_BINARY is defined and exists.